### PR TITLE
Fix for Mina-Signer exports.

### PR DESCRIPTION
**Pull request by shimkiv**

### CODE REVIEW

**Review by Trivo25 — APPROVED** [review]

File: src/mina-signer/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // this file is a wrapper for supporting types in both commonjs and esm projects
 
 import Client from './mina-signer.ts';
-export { type NetworkId } from './src/types.ts';
+import { NetworkId } from './src/types.ts';
 
-export = Client;
+export default Client;
```
Comment (mitschabaude) [inline src/mina-signer/index.d.ts:6]: was this change necessary?

Comment (shimkiv) [reply src/mina-signer/index.d.ts:6]: It does not impact functionality but since mina-signer is ESM project it removes error on that line if used to target CJS.
